package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@api-platform/admin",
-  "version": "3.4.8",
+  "version": "4.0.0",
```

Comment:
To revert, this will be handled by `yarn version` during the release process: https://github.com/api-platform/admin/blob/main/CONTRIBUTING.md#tag-a-new-version-contributors-only